x-pack/plugins/security_solution/public/management/pages/response_actions/index.tsx

```diff
@@ -4,14 +4,24 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { Switch } from 'react-router-dom';
+import { Redirect, Switch } from 'react-router-dom';
 import { Route } from '@kbn/kibana-react-plugin/public';
 import React, { memo } from 'react';
+import { ENDPOINTS_PATH, SecurityPageName } from '../../../../common/constants';
+import { useLinkExists } from '../../../common/links';
 import { MANAGEMENT_ROUTING_RESPONSE_ACTIONS_HISTORY_PATH } from '../../common/constants';
 import { NotFoundPage } from '../../../app/404';
 import { ResponseActionsListPage } from './view/response_actions_list_page';
 
 export const ResponseActionsContainer = memo(() => {
+  const canAccessResponseActionsHistoryNavLink = useLinkExists(
+    SecurityPageName.responseActionsHistory
+  );
+
+  if (!canAccessResponseActionsHistoryNavLink) {
```

Comment:
Same comment here as in the other PR. Maybe don't register the route at all and they will never reach this point.

Reply:
9075774cb8d914a80b8565e1416c8e387fa6a528